tests/src/Interop/StringMarshalling/UTF8/UTF8Test.cs

```diff
@@ -152,6 +155,13 @@ public static void TestUTF8StructMarshalling(string[] utf8Strings)
         }
         if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
          CompareWithUTF8Encoding();
+
+        SetStringInStruct(ref utf8Struct, "StructTestString");
```

Comment:
Care to embed a non-ascii character? Just so the test is really doing the right thing? Remember to use the [escaped unicode format](https://blogs.msdn.microsoft.com/csharpfaq/2004/03/12/what-character-escape-sequences-are-available/).